tools/version_mapping.go

```diff
@@ -102,9 +105,6 @@ func isSupported(network string, height int64, iter *VersionIterator) bool {
 		return v.nodeVersion == LatestMainnetVersion.nodeVersion
 	}
 
-	if height < V8.mainnet {
-		return v.nodeVersion == V7.nodeVersion
-	}
```

Comment:
If we now have this, what happens with all the switches we have where v1 ... v6 cases are not added, as types end up being the same? Shouldn't we uncomment them now? Maybe I am wrong, but just to understand